woodwork/table_accessor.py

```diff
@@ -457,6 +457,32 @@ def pop(self, column_name):
 
         return series
 
+    def drop(self, columns):
+        """Drop specified columns from a DataFrame.
+
+        Args:
+            columns (str or list[str]): Column name or names to drop. Must be present in the DataTable.
+
+        Returns:
+            (pd.DataFrame): DataFrame with the specified columns removed, maintaining Woodwork typing information.
+
+        Note:
+            This method is used for removing columns only. To remove rows with ``drop``, go through the
+            DataFrame directly instead of calling ``DataFrame.ww.drop``.
```

Comment:
Think it might be worth mentioning that after using the DataFrame method that reinitialization of Woodwork will be required if typing information is needed?

Reply:
Yes, that's a good idea. Changing to `...go through the DataFrame directly instead of calling ``DataFrame.ww.drop`` and then reinitialize Woodwork with DataFrame.ww.init`